resources/js/components/FileCard.vue

```diff
@@ -1,11 +1,15 @@
 <template>
     <card class="flex item-center overflow-hidden p-4">
-        <div
-            ref="fileIcon"
-            v-tooltip.click="formatedFileMeta"
-            class="flex items-center flex-no-shrink mr-4"
+        <a
```

Comment:
Better use `<icon type="download" view-box="0 0 24 24" width="16" height="16" />` to align with Nova UI